### PR TITLE
[ENH] add authorship agreement

### PR DESCRIPTION
**Pull request by Remi-Gau**
This is the equivalent of the current version of the google doc.

Not added in the TOC yaml as I am not sure where this will go.

### CODE REVIEW

**Review by shawnrhoads — COMMENTED** [review]
Content looks good to me!

@Remi-Gau brings up a great point about the table of contents ([_toc.yml](https://github.com/brainhackorg/brainhack_jupyter_book/blob/main/brainhack_book/_toc.yml)) though. 
At the meeting, we discussed making sure that the Jupyter Book (JB) links are static (i.e., a hyperlink in the NeuroView paper would permanently link to a page in the JB). 

Here are a few broad-strokes suggestions: 

I would recommend us implementing some heuristic for these pages. For example, we can use `neuroview_*-*.*` (e.g., `neuroview_authorship-agreement.md`) or `neuroview2021_*-*.*`  (e.g., `neuroview2021_authorship-agreement.ipynb`).

For the preprint/publication, we could start with something like this in the `_toc.yml`:
```
- part: Welcome
  chapters:
    - file: neuroview_index 
    - file: neuroview_overview
    - file: neuroview_geographical-distribution
    - file: neuroview_glossary

- part: Contributions
  chapters:
    - file: neuroview_contributors
    - file: neuroview_authorship-agreement

- part: Acknowledgements
  chapters:
    - file: neuroview_acknowledgements
```

Then, as the JB evolves, it would turn into something like this (which would keep all content and links frozen in time):
```
- file: index
- part: Welcome
  chapters:
    - file: index
    - file: overview
    - file: geographical-distribution
    - file: glossary

- part: Sections
  chapters: 
    - file: projects
    - file: training
    - file: open-science
    - file: reproducibility
    - file: community

 - part: Contributions
  chapters:
    - file: contributors
    - file: authorship-agreement

- part: Acknowledgements
  chapters:
    - file: neuroview_acknowledgements

- part: NeuroView 2021
  chapters:
    - file: neuroview_index
    - file: neuroview_overview
    - file: neuroview_geographical-distribution
    - file: neuroview_glossary 
    - file: neuroview_contributors
    - file: neuroview_authorship-agreement
    - file: neuroview_acknowledgements
```

Thoughts? We should probably open this up as an issue for broader discussion.

**Review by katjaq — APPROVED** [review]
Looks good and very detailed. Thank you for all the efforts @Remi-Gau ✨ 

One quick thought: I don't know if it can be formulated "better" – the last paragraph of the intro reads "Everybody in the community could join this group at any time to give feedback, help organizing or chair a meeting." --> Everybody in the community could join this group at any time to actively contribute content, help writing, creating figures, give feedback, help organizing or chair a meeting." And later in the process of course writing and figures were largely done, but that sounds natural :)

**Review by PeerHerholz — COMMENTED** [review]
+1 re the points @shawnrhoads outlined.

**Comment by shawnrhoads** [comment]
> 
> 
> +1 re the points @shawnrhoads outlined.

Great! We are getting ready to merge @htwangtw's phenomenal work for the acknowledgements and contributions pages (PR #14). Should we add a heuristic such as `neuroview2021-acknowledgements.md` before merging with main? Or (probably easier) we can just aim to make all these changes after everyone adds their work and right before the preprint submission (maybe the day before on 1/27/21). This would also give us some time to discuss more at our meeting this week.

What does everyone think?

**Comment by Remi-Gau** [comment]
> > +1 re the points @shawnrhoads outlined.
> 
> Great! We are getting ready to merge @htwangtw's phenomenal work for the acknowledgements and contributions pages (PR #14). Should we add a heuristic such as `neuroview2021-acknowledgements.md` before merging with main? Or (probably easier) we can just aim to make all these changes after everyone adds their work and right before the preprint submission (maybe the day before on 1/27/21). This would also give us some time to discuss more at our meeting this week.
> 
> What does everyone think?

yes to  `neuroview2021-acknowledgements.md`

and yes to rediscuss before final "push" (also because this is necessary to have the actual link in the preprint)

File: brainhack_book/authorship-agreement.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+The contributors spreadsheet resulting from this form was then used to
+help us divide authors into core-team and non-core-team members and to rank
+authors within the core team while non-core team members were listed
+alphabetically.
+
+Our motivation to divide contributors that way was to highlight authors who took
+a more involved part in the creation of the paper itself. 
+Everybody in the
+community could join this group at any time to give feedback, help organizing or
+chair a meeting.
```
Comment (Remi-Gau) [inline brainhack_book/authorship-agreement.md:35]:
```suggestion
Everybody in the community could join this group at any time to actively contribute content, help writing, creating figures, give feedback, help organizing or chair a meeting.
```

Comment (Remi-Gau) [reply brainhack_book/authorship-agreement.md:35]: @katjaq like this?

**Review by Remi-Gau — COMMENTED** [review]
update suggestion on phrasing

File: brainhack_book/authorship-agreement.md (hunk 1 above)
Comment (katjaq) [reply brainhack_book/authorship-agreement.md:35]: 🚀 great! :)

**Comment by Remi-Gau** [comment]
I suspect we can move @shawnrhoads [coment](https://github.com/brainhackorg/brainhack_jupyter_book/pull/13#pullrequestreview-574854814) in another issue and merge this for now?

**Review by shawnrhoads — APPROVED** [review]
All sounds good to me!

All that's left is adding this page to the toc so it builds with the book! Should we plan to eventually group this with the actual contributors page?

I'll open up a new issue re organization/naming of book!

**Review by PeerHerholz — APPROVED** [review]

**Comment by PeerHerholz** [comment]
Seems like we're ready for merging!

**Pull request merged**